lib/Dialect/Triton/Transforms/Combine.td

```diff
@@ -39,9 +39,14 @@ def CombineDotAddFRevPattern : Pat<
 //   Note: leave (sub %c0, %c0) canceling to ArithDialect
 //         (ref: ArithCanonicalization.td)
 defvar DefOverflow = ConstantEnumCase<Arith_IntegerOverflowAttr, "none">;
+
+def CopyDiscardableAttrs: NativeCodeCallVoid<
+        "$1.getOwner()->setDiscardableAttrs($0.getOwner()->getDiscardableAttrDictionary())">;
```

Comment:
so we blindly copy any discardable attributes? That doesn't seem safe. I think we should specifically handle the one we care about and know are correct